.github/workflows/label_new_pr.yml

```diff
@@ -9,26 +9,46 @@ on:
       - opened
 
 jobs:
+  # Documentation: https://docs.openverse.org/reference/ci_cd.html#get-changes
+  get-changes:
+    name: Get changes
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      changes: ${{ steps.paths-filter.outputs.changes }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Get changes
+        id: paths-filter
+        uses: ./.github/actions/get-changes
```

Comment:
IIRC this is run directly from the catalog and infrastructure repos, but I don't believe we sync the `get-changes` action (or even have logic in it for handling those repos). Will this continue to work for both of those repos?

Reply:
Ah yes, the `get-changes` action will need to be synced via our sync job.

Reply:
The alternative was for the Python to do incorporate the logic of `get-changes` but that would've been reinventing the wheel (and poorly). This is a workaround, yes, but it's a pretty clean (and functional) one.